site/docs/v1/tech/tutorials/integrate-flutter.md

```diff
@@ -0,0 +1,188 @@
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following file into `package.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/package.json %}
+```
+
+Now you need to install the dependencies in `package.json`.
+
+```shell
+npm install
+```
+
+Then copy and paste the following file into `setup.js`. This file uses the [FusionAuth API](/docs/v1/tech/apis/) to configure an Application, CORS and more to allow for easy integration.
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/setup.js %}
+```
+Then, you can run the setup script.
+```shell
+npm run setup
+```
+If you want, you can [login to your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your Flutter Application
+
+In order to create and set up the new {{technology}} app, run the following command. You'll want to be in a directory where the project should live.
+
+```shell
+flutter create fusionauth_demo
+```
+
+After the installation process completes, you will see that the `fusionauth_demo` directory contains all the Flutter starter app configuration. Open the project directory with the text editor of your choice.
+
+You can run your new project in the actual device or an emulator to confirm everything is working before you customize any code. Do so by running the following command in the project directory:
+
+```shell
+cd fusionauth_demo
+flutter run
+```
+
+After running this, you will get the list of emulators or devices in which you want to run the project. You can simply choose the one in which you want the code to run. Make sure to remain in this directory for the rest of this tutorial.
+
+However, to run the project on both iOS and Android together, you can use the following command:
+
+```shell
+flutter run -d all
+```
+
+The build process always takes a while the first time an app is built. After the successful build, you will get the boilerplate Flutter app running in your emulators.
+
+Now that you have a basic working application running, let's jump into the fun stuff: adding auth!
+
+## Integrating AppAuth
+
+[AppAuth](https://appauth.io/) is a popular OAuth package that can be used in both native and cross-platform mobile applications. In this project, you will be storing the access token using the secure storage package. Since such tokens allow your application to access protected resources such as APIs, you need to take care they are stored as securely as possible.
+
+Therefore, you need to install some dependencies to your project. For that, you need to open the `pubspec.yml` file in your project and add these dependencies:
+
+```yaml
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-flutter-dart/main/pubspec.yml %}
```

Comment:
```suggestion
{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-flutter-dart/main/pubspec.yaml %}
```

Reply:
After you update pubspec, do you have to restart flutter?

Reply:
I got this error message:

```
Error (Xcode): lib/main.dart:8:16: Error: The 'super-parameters' language feature is disabled for this library.
```

and per https://stackoverflow.com/questions/72197622/this-requires-the-super-parameters-language-feature-to-be-enabled had to change the minimum sdk version to 2.17 in pubspec.yaml

Reply:
Looks like there's a later version of app auth:

```
 flutter_appauth 0.9.2+6 (5.0.0 available)
```

We want to take this chance to upgrade this.